lib/display.ts

```diff
@@ -0,0 +1,289 @@
+
+  for (const from of fromList) {
+    result.push(from[0]);
+  }
+
+  return result.join(", ");
+}
+function formatFrom(fromList: string[][]): string {
+  const result: string[] = [];
+  let counter = 0;
+  const max = 3;
+  for (const localFrom of fromList) {
+    if (counter >= max) {
+      break;
+    }
+    counter += 1;
+
+    result.push(`  From: ${localFrom.join(" > ")}`);
+  }
+  if (fromList.length > max) {
+    result.push(`  and ${(fromList.length = max)} more...`);
+  }
+
+  return result.join(BREAK_LINE);
+}
+
+function formatFixedIn(issue: Issue): string | undefined {
+  if (!issue.fixInfo || !issue.fixInfo.nearestFixedInVersion) {
+    return undefined;
+  }
+
+  return chalk.bold.green(`  Fixed in: ${issue.fixInfo.nearestFixedInVersion}`);
+}
+
+function formatMetadataSection(
+  scanResult: ScanResult,
+  testResult: TestResult,
+): string {
+  const result: string[] = [];
+  result.push(formatMetadataLine("Organization:", testResult.org));
+
+  const packageManager = scanResult.identity.type;
+  result.push(formatMetadataLine("Package manager:", packageManager));
+
+  const target: ContainerTarget = scanResult.target as ContainerTarget;
+  const projectName = target.image;
+  const image = target.image.replace("docker-image|", "");
+  result.push(formatMetadataLine("Project name:", projectName));
+  result.push(formatMetadataLine("Docker image:", image));
+
+  return result.join(BREAK_LINE);
+}
+
+function formatMetadataLine(header: string, info: string = ""): string {
+  const padLength = 19;
+
+  return `${chalk.green(padding(header, padLength))} ${info}`;
+}
+
+function formatSummary(testResult: TestResult): string {
+  const depGraph: DepGraph = createFromJSON(testResult.depGraphData);
+  const pkgCount = depGraph?.getDepPkgs()?.length || 0;
+  const pathOrDepsText = `${pkgCount} dependencies`;
+  const testedInfoText = `Tested ${pathOrDepsText} for known issues`;
+  const vulnPathsText = formatVulnSummaryText(testResult.issues);
+  let summaryText = `${testedInfoText}, ${vulnPathsText}`;
+  if (testResult.issues.length === 0) {
+    summaryText = chalk.green(`✓ ${summaryText}`);
+  }
+
+  return summaryText;
+}
+
+function formatVulnSummaryText(issues: Issue[]): string {
+  if (issues.length > 0) {
+    return chalk.bold.red(`found ${issues.length} issues.`);
+  }
+
+  return "no vulnerable paths found.";
+}
+
+function getColor(severity: string): (text: string) => string {
+  let color: (text: string) => string;
+  switch (severity) {
+    case "low":
+      color = chalk.bold.blue;
+      break;
+    case "medium":
+      color = chalk.bold.yellow;
+      break;
+    case "high":
+      color = chalk.bold.red;
+      break;
+    default:
+      color = chalk.whiteBright;
+      break;
+  }
+
+  return color;
+}
+
+export function formatRemediations(res: TestResult) {
+  if (!res.docker || !res.docker.baseImageRemediation) {
+    return "";
+  }
+  const { advice, message } = res.docker.baseImageRemediation;
```

Comment:
`/test-dependencies` does not return this yet, could we sync with @darscan and @gitphill if this is an OK format to keep? Just noticing that `docker` is a very ecosystem-specific field for a generic /test endpoint.